code/modules/spells/spell_types/godhand.dm

```diff
@@ -110,3 +110,46 @@
 	M.Stun(40)
 	M.petrify()
 	return ..()
+
+/obj/item/melee/touch_attack/honk
+	name = "\improper silly touch"
+	desc = "Can you hear it? The pained laughter? The dull smack of a forehead against metal and glass? The incessant need to honk without end?"
+	catchphrase = "Lonk-Lonk-Lonk, FMR'shNK Cluw-NR!!!"
+	on_use_sound = 'sound/items/airhorn.ogg'
+	icon_state = "clown"
+	item_state = "clown"
+
+/obj/item/melee/touch_attack/honk/afterattack(atom/target, mob/living/carbon/user, proximity)
+	if(!proximity || !ishuman(target) || !iscarbon(user))
```

Comment:
proposal: make it so that using this on a slaughter demon turns them into a laughter demon

secondary proposal: make it so that using this on a bot (as in, a medibot, mulebot, secbot, etc.) turns them into a honkbot

tertriary proposal: make it so that using this on a cyborg (who isn't already a clownborg) switches them to the clown module.

Reply:
YES YES YES THESE ARE 3 AWESOME IDEAS